dev/conductor/core/lib/src/state.dart

```diff
@@ -7,15 +7,36 @@ import 'dart:convert' show JsonEncoder, jsonDecode;
 import 'package:file/file.dart' show File;
 import 'package:platform/platform.dart';
 
-import './globals.dart';
+import './globals.dart' as globals;
```

Comment:
nit: umm yeah I think this would cause variables and functions to be prefixed with globals.variables. just more typings in my opinion.

Reply:
That is a fair assessment.  My reasoning behind prefixing is to make it easier to find out where the variables are coming from.  It is also consistent with some of the other prefixes.

Reply:
umm looking at repository/start/next/state/clean.dart they seem to be not prefixed. Is our plan going forward to prefix them as well? I can RSLGTM